xcode/Subconscious/Shared/Components/Common/StoryComboView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  StoryComboView.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 24/9/2022.
+//
+
+import SwiftUI
+
+/// A story is a single update within the FeedView
+struct StoryComboView: View {
+    var story: StoryCombo
+    var action: (EntryLink, String) -> Void
+    
+    /// Sort entries by slug
+    /// (this should be factored out eventually)
+    func orderedSlugPair() -> (Slug, Slug) {
+        var entries = [story.entryA.slug, story.entryB.slug]
+        entries.sort(by: { $0.description < $1.description })
+        
+        return (entries[0], entries[1])
+    }
+    
+    /// Construct combined slug and default content, then open editor
+    func synthesize() {
+        let (first, second) = orderedSlugPair()
+        let content =
+            """
+            \(story.prompt)
+            
+            \(first.toSlashlink()) \(second.toSlashlink())
+            """
+        
+        guard let slug = Slug(formatting: "\(first) \(second)") else { return }
```

Comment:
Nit: formatting

```swift
guard let slug = Slug(formatting: "\(first) \(second)") else {
    return
}
```

Not blocking merge though.

Reply:
Fixed